optimizer_wrapper.rb

```diff
@@ -762,7 +763,7 @@ def self.parse_result(vrp, result)
         route = empty_route(vrp, vehicle)
         result[:routes] << route
       end
-      matrix = vrp.matrices.find{ |mat| mat.id == vehicle.matrix_id }
+      matrix = Models::Matrix.find_by_id(vehicle.matrix_id)
```

Comment:
In fact since we plan to have a proper "printer" for models, why don't we just add the matrix to the vehicles as a Models::Matrix so that we completely prevent these find_by logics (This might be out of scope of this dev, though, what do you think?)

Reply:
My expectation is that this change has to be shipped with the introduction of the models printer